scripts/regressions/dataset_scaler.py

```diff
@@ -0,0 +1,27 @@
+from pandas import DataFrame
+
+class Dataset_scaler:
```

Comment:
let's also add `get_scaled_dataset()` for retrieving the underlying scaled dataset and `get_scaled_value(value, column)` for getting the scaled value in the future

Reply:
and the name should probably be DatasetScaler? (correct me if I'm wrong)